platformics/codegen/templates/api/types/class_name.py.j2

```diff
@@ -356,7 +356,8 @@ Mutation types
 
 {% macro getInputFields(action, fields) -%}
     {%- for attr in fields %}
-        {%- if attr.type == "string" %}
+	{%- if attr.is_virtual_relationship %} {# TODO FIXME this prevents ID fields from 1:1 relationships see entities/schema/platformics.yaml #}
+        {%- elif attr.type == "string" %}
```

Comment:
This change makes sense - so no problem. However, I'm curious why there's no change in the generated output? This looks kindof like a no-op without that.

Reply:
Hmm that is sus let me check there should be

Reply:
I backed out this change and re-ran codegen and there's no change, so I'm more convinced this is a no-op.

Reply:
I just did the same I wonder if this issue was fixed by the PR that went in while I was working on this. There was a merge conflict I resolved. Let me remove this bit.

Reply:
Yes it seems like the new `getInputFields` fixes this, reverting.